recipes/libsbml/LICENSE.txt

```diff
@@ -0,0 +1,167 @@
+           Licensing and Distribution Terms for libSBML
```

Comment:
Please use the license(s) from the source tar instead of redistributing it here.

Reply:
Now done : 28e138c